src/components/Header.tsx

```diff
@@ -1,20 +1,128 @@
 import React, { useEffect, useState, useContext } from 'react';
 import clsx from 'clsx';
 import Link from 'next/link';
-import { useRouter } from 'next/router';
-
-import { GithubIcon } from '../components/icons/GithubIcon';
+import Router, { useRouter } from 'next/router';
+import { Dialog } from '@headlessui/react';
 import { Logo } from '../components/Logo';
-import { MobileNavigation } from '../components/MobileNavigation';
 import { Search } from '../components/Search';
 import { ThemeSelector } from '../components/ThemeSelector';
 import { ThemeContext } from '../contexts/ThemeContext';
 
-interface HeaderProps {
-  navigation: any;
-}
+export function NavPopover({ display = 'lg:hidden', className, ...props }) {
```

Comment:
Extract `NavPopover` to a separate component file.

Reply:
* Created a separate component

Reply:
Rename to something like `MobileNavbar`

Reply:
rename done